### DISCUSSION TITLE
23.10.16 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#30] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - @transactioneventlistener 적용
- 내일은 어떤 일을 하나요?
  - 트랜잭션 관련 테스트 케이스 추가

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
- 내일은 어떤 일을 하나요?
    - 코드 리뷰를 반영하여 코드 수정

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
* 내일은 어떤 일을 하나요?
  * 코드리뷰
  * 테스트 케이스 추가
